<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Настройки</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="styleSheet">
        <string notr="true">QTabBar::tab {
    border-bottom: None;
   /* border-top-left-radius: 4px;
    border-top-right-radius: 4px;*/
}


QTabBar::tab:selected {
   border: 1px solid #aaaaaa;
 border-bottom: 0px;
/*   background-color: #af0000;*/
  background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #dedeff, stop:1 #ffffff);
   color: #000000;
   padding-left: 20px;
   padding-right: 20px;
   width: 80px;
   height: 20px;
   border-top-left-radius: 6px;
   border-top-right-radius: 6px;
}
QTabBar::tab:!selected {
   border: 1px solid #aaaaaa;
   background-color: #ffffde;
   color: #515151;
   border-radius: 0 px;
   padding-left: 20px;
   padding-right: 20px;
   width: 80px;
   height: 20px;
   border-bottom: None;
   border-top-left-radius: 6px;
   border-top-right-radius: 6px;
  background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #ffffde, stop:1 #ffffff);
}
</string>
       </property>
       <property name="tabPosition">
        <enum>QTabWidget::North</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Подключение</string>
        </attribute>
        <widget class="QGroupBox" name="groupBox_8">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>201</width>
           <height>341</height>
          </rect>
         </property>
         <property name="title">
          <string>Настройки Финам по умолчанию</string>
         </property>
         <widget class="QPushButton" name="finam_hft">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>260</y>
            <width>151</width>
            <height>31</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Финам - HFT</string>
          </property>
         </widget>
         <widget class="QPushButton" name="finam_defaults">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>220</y>
            <width>151</width>
            <height>31</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Финам стандартные</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>20</y>
            <width>141</width>
            <height>151</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap>icons/register-icon.png</pixmap>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QPushButton" name="finam_demo">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>180</y>
            <width>151</width>
            <height>31</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Финам - демо</string>
          </property>
         </widget>
         <widget class="QPushButton" name="finam_mma">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>300</y>
            <width>151</width>
            <height>31</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Just2Trade MMA - демо</string>
          </property>
         </widget>
        </widget>
        <widget class="QGroupBox" name="groupBox_6">
         <property name="geometry">
          <rect>
           <x>230</x>
           <y>220</y>
           <width>341</width>
           <height>131</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="title">
          <string>Параметры соединения</string>
         </property>
         <widget class="QLabel" name="delay_label">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>121</width>
            <height>21</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Pf DinDisplay Pro</family>
            <pointsize>8</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string/>
          </property>
          <property name="text">
           <string>Частота</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="QLineEdit" name="rqdelay">
          <property name="geometry">
           <rect>
            <x>150</x>
            <y>20</y>
            <width>41</width>
            <height>21</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Pf DinDisplay Pro</family>
            <pointsize>10</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">border: 1px solid #999999;
font-weight: light;
font: 10pt &quot;Pf DinDisplay Pro&quot;;
text-align: center;
border-radius:3px;

</string>
          </property>
          <property name="inputMask">
           <string>999; </string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLabel" name="label_17">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>50</y>
            <width>121</width>
            <height>21</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Pf DinDisplay Pro</family>
            <pointsize>8</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string/>
          </property>
          <property name="text">
           <string>Таймаут сессии</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="QLabel" name="label_18">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>80</y>
            <width>121</width>
            <height>21</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Pf DinDisplay Pro</family>
            <pointsize>8</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string/>
          </property>
          <property name="text">
           <string>Таймаут заявки</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="QLineEdit" name="line_sesstimeout">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>150</x>
            <y>50</y>
            <width>41</width>
            <height>21</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Pf DinDisplay Pro</family>
            <pointsize>10</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Интервал времени, в течении которого коннектор&lt;br/&gt;в случае ошибок связи будет выполнять попытки&lt;br/&gt;переподключения к серверу без повторной закачки&lt;br/&gt;информации о доступных периодах исторической&lt;br/&gt;ценовой информации.&lt;/p&gt;&lt;p&gt;Значение по умолчанию равно 120 секундам.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="styleSheet">
           <string notr="true">border: 1px solid #999999;
font-weight: light;
font: 10pt &quot;Pf DinDisplay Pro&quot;;
text-align: center;
border-radius:3px;

</string>
          </property>
          <property name="text">
           <string>10</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLineEdit" name="line_rqtimeout">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>150</x>
            <y>80</y>
            <width>41</width>
            <height>21</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Pf DinDisplay Pro</family>
            <pointsize>10</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">border: 1px solid #999999;
font-weight: light;
font: 10pt &quot;Pf DinDisplay Pro&quot;;
text-align: center;
border-radius:3px;

</string>
          </property>
          <property name="text">
           <string>5</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLabel" name="label_19">
          <property name="geometry">
           <rect>
            <x>200</x>
            <y>50</y>
            <width>101</width>
            <height>21</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Pf DinDisplay Pro</family>
            <pointsize>8</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>секунд</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="QLabel" name="label0">
          <property name="geometry">
           <rect>
            <x>200</x>
            <y>80</y>
            <width>101</width>
            <height>21</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Pf DinDisplay Pro</family>
            <pointsize>8</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>секунд</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="QLabel" name="label1">
          <property name="geometry">
           <rect>
            <x>200</x>
            <y>20</y>
            <width>101</width>
            <height>21</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Pf DinDisplay Pro</family>
            <pointsize>8</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>миллисекунд</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </widget>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="geometry">
          <rect>
           <x>230</x>
           <y>120</y>
           <width>341</width>
           <height>91</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="title">
          <string>Договор</string>
         </property>
         <widget class="QLineEdit" name="login">
          <property name="geometry">
           <rect>
            <x>150</x>
            <y>20</y>
            <width>151</width>
            <height>21</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Pf DinDisplay Pro</family>
            <pointsize>10</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">border: 1px solid #999999;
font-weight: light;
font: 10pt &quot;Pf DinDisplay Pro&quot;;
text-align: center;
border-radius:3px;

</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLabel" name="login_label">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>20</y>
            <width>81</width>
            <height>21</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Pf DinDisplay Pro</family>
            <pointsize>8</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string/>
          </property>
          <property name="text">
           <string>Логин</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="QLabel" name="password_label">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>50</y>
            <width>81</width>
            <height>21</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Pf DinDisplay Pro</family>
            <pointsize>8</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string/>
          </property>
          <property name="text">
           <string>Пароль</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="QLineEdit" name="password">
          <property name="geometry">
           <rect>
            <x>150</x>
            <y>50</y>
            <width>151</width>
            <height>21</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Pf DinDisplay Pro</family>
            <pointsize>10</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">border: 1px solid #999999;
font-weight: light;
font: 10pt &quot;Pf DinDisplay Pro&quot;;
text-align: center;
border-radius:3px;

</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </widget>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="geometry">
          <rect>
           <x>230</x>
           <y>10</y>
           <width>341</width>
           <height>91</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="inputMethodHints">
          <set>Qt::ImhNone</set>
         </property>
         <property name="title">
          <string>Торговый сервер</string>
         </property>
         <widget class="QLineEdit" name="ip">
          <property name="geometry">
           <rect>
            <x>150</x>
            <y>20</y>
            <width>125</width>
            <height>20</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Pf DinDisplay Pro</family>
            <pointsize>10</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">border: 1px solid #999999;
font-weight: light;
font: 10pt &quot;Pf DinDisplay Pro&quot;;
text-align: center;
border-radius:3px;

</string>
          </property>
          <property name="inputMethodHints">
           <set>Qt::ImhExclusiveInputMask</set>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLabel" name="ip_label">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>20</y>
            <width>81</width>
            <height>21</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Pf DinDisplay Pro</family>
            <pointsize>8</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Адрес</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="QLabel" name="port_label">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>50</y>
            <width>81</width>
            <height>21</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Pf DinDisplay Pro</family>
            <pointsize>8</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Порт</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="QLineEdit" name="port">
          <property name="geometry">
           <rect>
            <x>150</x>
            <y>50</y>
            <width>113</width>
            <height>21</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Pf DinDisplay Pro</family>
            <pointsize>10</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">border: 1px solid #999999;
font-weight: light;
font: 10pt &quot;Pf DinDisplay Pro&quot;;
text-align: center;
border-radius:3px;

</string>
          </property>
          <property name="inputMethodHints">
           <set>Qt::ImhDigitsOnly|Qt::ImhPreferNumbers</set>
          </property>
          <property name="inputMask">
           <string>99999; </string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="maxLength">
           <number>5</number>
          </property>
         </widget>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="button_cancel_settings">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">padding:5 px;
background-color: #ff3e3e;
color: #ffffff;</string>
       </property>
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_save_settings">
       <property name="styleSheet">
        <string notr="true">background-color: #3eaf3e;
color: #ffffff;</string>
       </property>
       <property name="text">
        <string>Сохранить настройки</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_cancel_settings</sender>
   <signal>pressed()</signal>
   <receiver>Form</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>153</x>
     <y>413</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
